examples/titanic_example/assets/function_random_forest/predict/Dockerfile

```diff
@@ -2,7 +2,7 @@
 FROM ghcr.io/substra/substra-tools:0.20.0-nvidiacuda11.8.0-base-ubuntu22.04-python3.9
 
 # install dependencies
-RUN pip3 install pandas numpy 'scikit-learn==0.24.2' pillow scipy keras
```

Comment:
Surprising to see Keras here 😅

Reply:
Same 😅 
It's not exactly lightweight either ^^